src/Aspire.Hosting.ValKey/README.md

```diff
@@ -0,0 +1,39 @@
+# Aspire.Hosting.ValKey library
```

Comment:
(this applies everywhere)

It looks like the stylized version of this project is `Valkey` (lower case `k`). https://github.com/valkey-io/valkey#what-is-valkey

Reply:
Corrected